Makefile

```diff
@@ -27,9 +27,19 @@ proto:
 crd:
 	@ cd client && ./hack/update-crd.sh
 
+.PHONY: lint
+# Run golangci-lint
+lint:
+	golangci-lint run
+
 
 # Include release-tools
 
 CMDS=csi-snapshot-metadata
 
 include release-tools/build.make
+
+.PHONY: test
+# Extend the test target to include lint
+test: lint
+	$(MAKE) -f release-tools/build.make test
```

Comment:
This line should not be necessary. All `test` targets from build.make will execute anyway.